src/targets/awsLambdaLayer.ts

```diff
@@ -171,23 +171,24 @@ export class AwsLambdaLayerTarget extends BaseTarget {
 
     await withTempDir(
       async directory => {
-        const git = simpleGit(directory);
         this.logger.info(
           `Cloning ${remote.getRemoteString()} to ${directory}...`
         );
-        await git.clone(remote.getRemoteStringWithAuth(), directory);
-
-        if (!isDryRun()) {
-          await this.publishRuntimes(
-            version,
-            directory,
-            awsRegions,
-            artifactBuffer
-          );
-          this.logger.debug('Finished publishing runtimes.');
-        } else {
-          this.logger.info('[dry-run] Not publishing new layers.');
-        }
+        await createGitClient('.').clone(remote.getRemoteStringWithAuth(), directory);
```

Comment:
Why are we not doing `createGitClient(directory)` instead of `createGitClient('.')`??